foundation/foundation.css

```diff
@@ -0,0 +1,133 @@
+body {
+    font-family: Arial,Helvetica,Verdana,sans-serif;
+    font-size: 15px;
+    line-height: 22px;
+    color: #666666;
+}
+
+h1 {
+    font-size: 32px;
+    line-height: 36px;
+    color: #333333;
+    padding-bottom: 18px;
+}
+
+h2 {
+    font-size: 24px;
+    line-height: 22px;
+    color: #333333;
+    padding-bottom: 14px;
```

Comment:
Here you use padding, but in special stylesheet you use margin :S.

Reply:
I think that's why I called it special, I had to use margin because the function is separate the color background of the boxes ;)... not the same end